chemicaltoolbox/rdock/rbcavity.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="rbcavity" name="rDock cavity definition" version="0.1">
+    <description>- generate the active site definition needed for rDock docking</description>
+    <requirements>
```

Comment:
Would a macro make sense here? For the requirement, the citation and the version number?

Reply:
Use macro: https://github.com/tdudgeon/galaxytools/commit/3fe85518879a9a987112f46eb10d2a028575d973